src/components/app/details/appDetails/DeploymentStatusDetailModal.tsx

```diff
@@ -3,6 +3,7 @@ import { ReactComponent as Close } from '../../../../assets/icons/ic-close.svg'
 import DeploymentStatusDetailBreakdown from './DeploymentStatusBreakdown'
 import { DeploymentStatusDetailModalType } from './appDetails.type'
 import { useHistory, useParams } from 'react-router-dom'
+import './appDetails.scss'
```

Comment:
Check if there's no impact of this SCSS inclusion

Reply:
Done